tests/Aspire.Oracle.EntityFrameworkCore.Tests/ConformanceTests.cs

```diff
@@ -66,7 +66,7 @@ protected override void SetHealthCheck(OracleEntityFrameworkCoreSettings options
         => options.DisableHealthChecks = !enabled;
```

Comment:
Can you add tests for the tracing. They need to be declared at each component level. For example:

https://github.com/dotnet/aspire/blob/9fab2f6bd66866fb210f3b6c1f6347a73c9ab57a/tests/Aspire.Npgsql.EntityFrameworkCore.PostgreSQL.Tests/ConformanceTests.cs#L134-L137

Reply:
Will do

Reply:
From what I can see this will require adding the TestContainers.Oracle nuget package to the internal package feed as https://github.com/dotnet/aspire/issues/3063 hasn't covered Oracle yet. Happy to give that a go.

Reply:
It also seems that the current .Net Testcontainers is using the XE image, there isn't the equivalent of the Java one having the newer Oracle free module. It also locks some of the config behind private methods that would swapping out the image to support the same image as Hosting uses. A bit of pondering required on the best approach.

Reply:
Added and also enabled Testcontainers for Oracle as part of it. Turns out the base Testcontainers instance works fine.